index.js

```diff
@@ -81,19 +92,23 @@ let Rosnodejs = {
    * @return {Promise} resolved when connection to master is established
    */
   initNode(nodeName, options) {
+    options = options || {};
+    if (options.anonymous) {
+      nodeName = _anonymizeNodeName(nodeName);
+      console.log('anonymous node name ' + nodeName);
```

Comment:
Just an FYI, this is the only spot in the file we `console.log()`, or any logging at all for that matter. Will this be replaced by your `rosnodejs` logging in the future?